apps/condo/migrations/20220630110118-0141_payment_order_paymenthistoryrecord_order.js

```diff
@@ -0,0 +1,34 @@
+// auto generated by kmigrator
```

Comment:
Please create the new migration file, cause of my [migration](https://github.com/open-condo-software/condo/pull/1835/files#diff-d88e4b725d67b36fa324ee22d110638e68743f68fcce0358782cd964c40ac1ea), sorry :)

Reply:
Done